src/styles/LandingPage.module.less

```diff
@@ -6,23 +6,51 @@
     width: 100vw;
 }
 
+/*
+* US map container
+*/
+.LandingPageMapImageContainer   {
+    position: relative;
+    text-align: center;
+    width: 100vw;
+}
 
 .ImageStyle { 
     filter: brightness(0.5);
     width: 100%;
 }
 
+// Use padding to shift US map image off center
+.MapImageStyle{
+    height: auto;
+    width: 118%;
+    transform: translate(-4%, 0%);
+    padding-left: 24.9%;
+}
+
+// Needs to be changed
 .LandingPageOverlayText { 
-    z-index: 100;
     position: absolute;
-    color: white;
-    font-size: calc(1.5vw + 8px);
+    display: inline-block;
+    margin: auto;
+    z-index: 100;
+    top: 7%;
+    right: 0;
+    left: 0;
+    width: 75.3%;
+    color: #373737;
+    display: inline-block;
+    font-size: calc(2.1vw + 26.1px); //arbitrary
     font-weight: 900;
```

Comment:
Somehow the font in Zeplin looks "bolder". We might need to play around with the font-style or font-family more to get what we want.

Reply:
@minkyoseo Based on the preview you see do you think the text is "bold enough"?

Reply:
Can you try switching the font size to REM and EM units? 

https://css-tricks.com/confused-rem-em/

Reply:
Sizes are no `rem` in Zeplin for easy carryover.

Reply:
Do we want to make the text size dynamic or is that planned for later down the line?

Reply:
I switched font-size and line-height to rem